pkg/apis/componentconfig/register.go

```diff
@@ -44,8 +44,6 @@ func Resource(resource string) schema.GroupResource {
 
 func addKnownTypes(scheme *runtime.Scheme) error {
 	// TODO this will get cleaned up with the scheme types are fixed
-	scheme.AddKnownTypes(SchemeGroupVersion,
-		&KubeSchedulerConfiguration{},
-	)
+	scheme.AddKnownTypes(SchemeGroupVersion)
```

Comment:
you can completely remove this function call and just write `// TODO: All structs in this package are about to be moved out, so nothing should be registered here as this API group is going to be removed soon.`